tokenserver/scripts/purge_old_records.py

```diff
@@ -116,6 +136,9 @@ def delete_service_data(config, service, user, timeout=60):
     secret = tokenlib.get_derived_secret(token, secret=node_secrets[-1])
     endpoint = pattern.format(uid=user.uid, service=service, node=user.node)
     auth = HawkAuth(token, secret)
+    if settings.dryrun:
+        logging.info("Deleting", endpoint, auth, timeout)
```

Comment:
logger?

Reply:
Also missing the format specifiers
```suggestion
        logger.info("Deleting endpoint: %s auth: %s", endpoint, auth)
```
(not sure we even want the full auth output'd)